GettingStarted.md

```diff
@@ -43,13 +46,13 @@
 	8. now_playing_max_lines: Max lines displayed on Now Playing Screen
 	9. draw_now_playing_menu: Show navigation menu on Now Playing screen
 	10. boot_device_name: Customize the name displayed on boot
-    11. display_timeout: Turn off display [s]
-    12. sleep_after: Enter deep sleep [s]
-	- More configuration can be done in esphome/esphome-remote-base.yaml
-		1. Customize **Colors**
-		2. Customize **Fonts**
-			- If you edit the font size you have to update both the font size and the corresponding font_size in Globals
-		3. Customize **Globals**
+	11. display_timeout: Turn off display [s]
+	12. sleep_after: Enter deep sleep [s]
+- More configuration can be done in esphome/esphome-remote-base.yaml
+	1. Customize **Colors**
+	2. Customize **Fonts**
+		- If you edit the font size you have to update both the font size and the corresponding font_size in Globals
+	3. Customize **Globals**
 
 4. Update the **Custom Components** in the yaml to match your config
 	1. Set your speaker media player names under speaker_group_component
```

Comment:
Maybe mention that TV setup is optional? Like if you don't have a roku tv you can just delete the soundbar and create an empty vector for the TV.
IMO this could throw some people of if they're no programmers and have no TV.

Reply:
updated, good idea